Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt
+++ CMakeLists.txt	2020-02-11 17:42:22.013323516 +0100
@@ -87,7 +87,7 @@
     endif ()
 elseif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
     # Clang++
-    set(CMAKE_CXX_FLAGS "-Wall -pedantic -Wextra -std=c++11 -stdlib=libc++ ${CMAKE_CXX_FLAGS}")
+    set(CMAKE_CXX_FLAGS "-Wall -pedantic -Wextra -std=c++11 ${CMAKE_CXX_FLAGS}")
 elseif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
     # Microsoft Visual C++
 else ()
